bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -58,7 +58,7 @@
     <h1>{{ self.page_title() }}</h1>
   </header>
 
-  {% if switch('firefox-mr1-launch') %}
+  {% if not switch('firefox-mr1-launch') %}
```

Comment:
Ah, looks like this snuck in during dev (my bad I should have caught this before merging). Thankfully a test failure caught it! https://gitlab.com/mozmeao/www-config/-/jobs/1549619134